generators/app/templates/documentation.js

```diff
@@ -0,0 +1,158 @@
+const config = require('./config');
+
+const port = config.common.api.port || 8080;
+
+module.exports = {
+  openapi: '3.0.1',
+  info: {
+    version: '0.1.0',
+    title: '<%= projectName %>',
+    description: '<%= projectDescription %>',
+    termsOfService: '',
+    contact: {
+      name: 'Wolox',
+      email: 'tls@wolox.com.ar',
+      url: 'https://www.wolox.com.ar/'
+    },
+    license: {
+      name: 'MIT'
+    }
+  },
+  servers: [
+    {
+      url: `http://localhost:${port}/`,
+      description: 'Local server'
+    },
+    {
+      url: 'https://api_url_testing',
+      description: 'Testing server'
+    }
+  ],
+  security: [],
+  tags: [
+    {
+      name: 'CRUD operations'
+    }
+  ],
+  paths: {
+    '/users': {
+      get: {
+        tags: ['CRUD operations'],
+        description: 'Get users',
+        operationId: 'getUsers',
+        parameters: [
+          {
+            name: 'page',
+            in: 'query',
+            schema: {
+              type: 'integer',
+              default: 1
+            },
+            required: false
+          }
+        ],
+        responses: {
+          200: {
+            description: 'Users were obtained',
+            content: {
+              'application/json': {
+                schema: {
+                  $ref: '#/components/schemas/Users'
+                }
+              }
+            }
+          }
+        }
+      },
+      post: {
+        tags: ['CRUD operations'],
+        description: 'Create user',
+        operationId: 'createUser',
+        parameters: [],
+        requestBody: {
+          content: {
+            'application/json': {
+              schema: {
+                $ref: '#/components/schemas/User'
+              }
+            }
+          },
+          required: true
+        },
+        responses: {
+          200: {
+            description: 'New user was created'
+          },
+          400: {
+            description: 'Invalid parameters',
+            content: {
+              'application/json': {
+                schema: {
+                  $ref: '#/components/schemas/Error'
+                },
+                example: {
+                  message: 'User´s email already exists',
+                  internal_code: 'invalid_parameters'
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  components: {
+    schemas: {
+      userId: {
+        type: 'integer',
+        example: 7
+      },
+      username: {
+        type: 'string',
+        example: 'tom99'
+      },
+      userEmail: {
+        type: 'string',
+        example: 'tom.engels@wolox.com.ar'
+      },
+      User: {
+        type: 'object',
+        properties: {
+          id: {
+            $ref: '#/components/schemas/userId'
+          },
+          username: {
+            $ref: '#/components/schemas/username'
+          },
+          email: {
+            $ref: '#/components/schemas/userEmail'
```

Comment:
I like the example, but i don't know if i want that in bootstrap, maybe we can move examples to techs guides and generics configs here, what do you think ?

Reply:
I consider that is not too clear that an example of the documentation can be there. People clone the repo and start making code. I prefer to have an example here, it would save people time using it.

Reply:
In that way you are creating a dependency